src/main/java/com/example/application/backend/repositories/RoleRepository.java

```diff
@@ -3,5 +3,14 @@
 import com.example.application.backend.entities.RoleEntity;
 import org.springframework.data.jpa.repository.JpaRepository;
 
+/**
+ * DESCRIPTION
+ *
+ * @author  Monika Martius and Jessica Reistel
```

Comment:
Sollte hier @ author Sabrine Gamdou , @ since 05.01.2021 , @ version 2.0 , @ lastUpdated Monika Martius and Jessica Reistel stehen

Reply:
Da stand tatsächlich vorher noch kein Kommentar, deswegen keine Ahnung wann es erstellt wurde. Ist aber @version 2.0 und @lastUpdated Monika Martius, Jessica Reistel

Reply:
In der History von github kann man das finden

Reply:
Ah achso, danke